backend/src/app.module.ts

```diff
@@ -10,6 +12,11 @@ import { PrismaModule } from "./prisma/prisma.module";
 import { MatchModule } from "./match/match.module";
 import { validate, configuration } from "./config";
 import { JwtAccessGuard } from "./auth/guard";
+import { RedisCacheModule } from "./cache/redisCache.module";
+
+const generateEmailDefaultFrom = (name: string, email: string): string => {
+  return '"' + name + '"' + "<" + email + ">";
+};
```

Comment:
can move this function to a file under the `utils/mail` folder? to try and keep this file as clean as possible